core/zip/src/RZip.cxx

```diff
@@ -132,7 +132,6 @@ void R__zipPrecisionCascade(int *srcsize, char *src, int *tgtsize, char **tgts,
    }
 
    auto content = reinterpret_cast<ROOT::Internal::PrecisionCascadeConfigArrayContent*>(configarray);
-   (void) configsize;
```

Comment:
`configsize` is only use in the assert.  Without this line, the compiler will complain about unused parameter in Release mode (where the assert is replaced with 'nothing').